packages/overlay/src/overlay-trigger.ts

```diff
@@ -109,21 +133,25 @@ export class OverlayTrigger extends LitElement {
         );
 
         this.dispatchEvent(overlayCloseEvent);
+        // overlayRoot.onOverlayClose(overlayCloseEvent);
```

Comment:
Woops, will review whether this should come back or be removed.

Reply:
I cannot remember why that was needed.